src/MarketingPageAcceptanceTests/MarketingPageAcceptanceTests.Actions/Pages/EditFeatures.cs

```diff
@@ -8,32 +9,34 @@ namespace MarketingPageAcceptanceTests.Actions.Pages
 {
     public sealed class EditFeatures : PageAction
     {
-        private string randomString = "";
 
         public EditFeatures(IWebDriver driver, ITestOutputHelper helper) : base(driver, helper)
         {
         }
 
         /// <summary>
-        /// Add random characters to a text field
+        /// Add random characters to a random text field
         /// </summary>
         /// <param name="characterCount">The number of characters to add</param>
         public string AddTextToFeature(int characterCount = 100)
         {
-            var characters = @"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890!£$%^&*(){}:@~<>?|[];#,./";            
-
-            var random = new Random();
-            for(int i = 0; i<= characterCount; i++)
-            {
-                randomString += characters[random.Next(characters.Length)];
-            }
-
-            driver.FindElements(pages.EditFeatures.FeatureText).First().Clear();
-            driver.FindElements(pages.EditFeatures.FeatureText).First().SendKeys(randomString);
-
+            var randomTextField = GetRandomTextField();
+            string randomString = random.GetRandomString(characterCount);
+            randomTextField.Clear();
+            randomTextField.SendKeys(randomString);
             return randomString;
         }
 
+        /// <summary>
+        /// 
```

Comment:
Empty???

Reply:
It was left empty to prevent unnecessary comments taking up too much space. The purpose of the method is in its name and '<returns>' description.